docs/howto/features/exam.md

```diff
@@ -0,0 +1,38 @@
+# Set up a hub for an exam
+
+We provide support for using a JupyterHub in a *controlled* physical
+space for exams. This is an extra paid feature where we charge hourly
+for responsiveness.
+
+This page documents what we do to prep, based on our prior experiences.
+
+1. Make sure the exact dates and times of the exam are checked well in
+   advance, and we have enough engineering coverage during this time period.
+   Engineers should also *test* their access to the infrastructure and the
+   hub beforehand, to make sure they can fix issues if needed.
+
+2. For the duration of the exam, all user pods must have a
+   [guaranteed quality of service class](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/).
+   In practice, this means we have memory & cpu requests set to be the same
+   as guarantees. This is to ensure equity - no user should get more or less
+   resources than any other. It also improves reliability.
+
+   This usually increases cost too, so should be done no more than 12h before
+   the start of the exam. It should be reverted back soon after the exam
+   is done.
+
+3. The instructor running the exam should test out their exam on the hub,
```

Comment:
As an SRE my action is unclear for this point - who is responsible for facilitating this and signing this off - is it engineering, communities & partnerships, or the community themselves.